draft-ietf-mls-architecture.md

```diff
@@ -1450,10 +1450,10 @@ the MLS group.  The application would need to provide a synchronization
 mechanism so that the clients' state remained consistent across changes to the
 MLS group.
 
->**RECOMMENDATION:** Avoid sharing cryptographic state between clients to
-> improve resilience against compromises.  An attacker could use one compromised
-> device to establish ownership of a state across other devices and reduce the
-> ability of the user to recover.
+>**RECOMMENDATION:** Only use secret state synchronization mechanisms that have
+> been widely reviewed and are known not to impair the security guarantees of
+> MLS. When this is not possible, avoid trying to synchronize state between
```

Comment:
I don't think this first sentence is an improvement.

Reply:
Multiple people have expressed need to share secret state, there are multiple drafts on the topic. You want to say "we recommend against this full stop" ?

Reply:
No, that's not correct. I don't have a strong opinion on key sharing one way or the other. Rather, this text went through WGLC and what it says is that you shouldn't share state because one device might be compromised and impair the ability of the user to recover. This threat is not addressed by having a secure way to share keys, and so the text you propose is a non sequitur.